stockflux-securities-master/src/reducers/securitiesTableReducer.js

```diff
@@ -0,0 +1,32 @@
+import { TableState } from "../enums";
+
+export const initialTableState = {
+  fetchStatus: TableState.LOADING,
+  hasErrors: false,
+  messages: []
+};
+
+export function securitiesTableReducer(state, action) {
+  switch (action.type) {
+    case "loading":
```

Comment:
I think you can do this:
`case TableState.LOADING:`

Reply:
yep good point, changed now